examples/wursthall/models/src/order_item_details.py

```diff
@@ -50,22 +51,15 @@ def execute(
     menu_item_details_table_name = context.table("src.menu_item_details")
 
     df_customers = context.fetchdf(
-        f"""
-        SELECT
-            id AS customer_id,
-            register_ds
-        FROM {customer_details_table_name}
-        WHERE
-            register_ds <= '{to_ds(end)}'
-        """
+        exp.select("id AS customer_id", "register_ds", copy=False)
+        .from_(customer_details_table_name, copy=False)
+        .where(f"register_ds <= '{to_ds(end)}'", copy=False),
+        quote_identifiers=True,
```

Comment:
Can we keep these as raw SQL? I don't want to leave the impression on new users that they have to write/learn sqlglot expressions.

Reply:
i think the tricky part is that this is a multi dialect project, so george can't just write raw sql because of differences between dialects, what he could do is do

parse_one(raw_sql, dialect="") which washes away the differences

Reply:
Exactly what Toby said -- happy to use parse_one instead if that makes it more clear.

Reply:
yea change it to parse_one, and leave a comment saying why you did it

Reply:
Changed.